music21/midi/translate.py

```diff
@@ -2590,6 +2616,7 @@ def streamHierarchyToMidiTracks(
     *,
     acceptableChannelList=None,
     addStartDelay=False,
+    encoding: str = 'utf-8',
```

Comment:
If you're not able to fix the mypy errors that adding `encoding: str = 'utf-8'` caused, just change this to `encoding='utf-8'` and the mypy errors will go away.  I'll be able to add typing to that anyhow.

Once the mypy errors are gone, I'm happy to merge -- the tiny drop in coverage is very small.